scripts/environment.sh

```diff
@@ -61,22 +61,10 @@ export CAPM3_VERSION=${CAPM3_VERSION:-"v1beta1"}
 export M3PATH=${M3PATH:-"${HOME}/go/src/github.com/metal3-io"}
 export CAPM3_LOCAL_IMAGE="${CAPM3PATH}"
 
+export PATH=$PATH:$HOME/.krew/bin
 # Upgrade test environment vars and config
 if [[ ${GINKGO_FOCUS:-} == "upgrade" ]]; then
-    export CAPI_VERSION="v1alpha4"
-    export CAPM3_VERSION="v1alpha5"
-    # Ironic and BMO images to start from. They will then upgrade to main/latest
-    export IRONIC_TAG="capm3-v0.5.5"
-    export BAREMETAL_OPERATOR_TAG="capm3-v0.5.5"
-fi
-# Override project infra vars that point to
-# the current branch to build capm3 image and crds
-if [[ "${CAPM3_VERSION}" == "v1alpha5" ]]; then
-    export CAPM3BRANCH="release-0.5"
-    # This var is set in project infra to use the current repo location for
-    # building CAPM3 image while upgrade needs an old version
-    unset CAPM3_LOCAL_IMAGE
-    export CAPM3PATH="${M3PATH}/cluster-api-provider-metal3"
+export NUM_NODES=${NUM_NODES:-"5"} 
```

Comment:
I am not understanding why we would need all of these

Reply:
capi test add scale up check after upgrade and for this we need additional bmh https://github.com/kubernetes-sigs/cluster-api/blob/4d8c88f24430d4fd15109e53fba57ad2acf5ddc8/test/e2e/clusterctl_upgrade.go#L393

Reply:
I am not sure got this right